integration/sawtooth_integration/tests/test_intkey_smoke.py

```diff
@@ -89,7 +89,10 @@ def post_and_verify(self, batch, how_many_updates):
         batch = IntkeyMessageFactory().create_batch(batch)
         LOGGER.info('Posting batch')
         _post_batch(batch)
-        time.sleep(1)
+
+        while (how_many_updates + 1) > _get_block_num():
```

Comment:
If you waited for batch status to change instead, could we avoid this sleep(1) in most runs?

If _get_block_num() stops changing (due to something being broken), does this produce a reasonable test failure due to test runner timeouts?